src/main/java/com/google/sps/classes/UserAuthenticationData.java

```diff
@@ -20,12 +20,14 @@
 public final class UserAuthenticationData {
 
   private final boolean loggedIn;
-  private final String authenticationUrl; // URL to redirect to to login or logout page.
   private final String email;
+  private final boolean isUserRegistered;
+  private final String authenticationUrl; // URL to redirect to to login or logout page.
```

Comment:
nit: Remove extra 'to'
```suggestion
  private final String authenticationUrl; // URL to redirect to login or logout page.
```

Reply:
Done!